src/components/Landing/Illustrations/Conditions.jsx

```diff
@@ -0,0 +1,145 @@
+const Conditions = () => {
```

Comment:
better to always follow the structure of the project
we don't have Landing/Illustration/.....jsx
everything is 

Component/Component.jsx

Let's make sure we are not inventing a new style half way through the project